src/test/fuzz/validation_load_mempool.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <chainparamsbase.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <test/util/setup_common.h>
+#include <txmempool.h>
+#include <util/time.h>
+#include <validation.h>
+
+#include <cstdint>
+#include <vector>
+
+namespace {
+FuzzedFileProvider* fuzzed_file_provider_ptr = nullptr;
+
+FILE* fuzzed_fopen(const fs::path&, const char*)
+{
+    return fuzzed_file_provider_ptr->open();
+}
+} // namespace
```

Comment:
Could avoid this namespace by making `fuzzed_fopen` a lambda with smallest scope possible?

Reply:
Good point! Now addressed.